templates/jlink-interim-build-template.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    description: Template to produce an imagestream and buildconfig for a Jlinked intermediate image
+  name: s2i-build-template
+parameters:
+- name: APP_NAME
+  description: Name of the application
+  required: true
+- name: GIT_URI
+  description: Git repository URL
+  required: true
+- name: GIT_REF
+  description: Git branch or tag to build
+  required: true
+- name: CONTEXT_DIR
+  description: Context directory which is a directory within git repo
+  required: true  
+- name: BUILDER_IMAGE
+  description: Name of the S2I builder image
+  required: true
+- name: OUTPUT_IMAGE_TAG
+  description: Tag for the output image
+  required: true
+- name: S2I_ENABLE_JLINK
```

Comment:
This would not need to be parameterized, because it will always be true for this workflow

Reply:
ok